modules/btcpay/btcpay.php

```diff
@@ -411,12 +411,6 @@ public function hookPaymentOptions(): array
 				return [];
 			}
 
-			// Prepare smarty
-			$this->context->smarty->assign([
```

Comment:
You could keep that and the corresponding HTML template at the payment_option.tpl if you just call the global getPaymentMethods and not split it between LN and on-chain, just list the returned payment options? Or do you think it does not need to be there anyway?

Reply:
I was unable to get any text to show. Will fix at a later time. 😁